managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/cache/RangeEntryCacheManagerImpl.java

```diff
@@ -106,12 +106,12 @@ public void removeEntryCache(String name) {
         entryCache.clear();
 
         if (log.isDebugEnabled()) {
-            log.debug("Removed cache for {} - Size: {} -- Current Size: {}", name, size / MB, currentSize.get() / MB);
+            log.debug("Removed cache for {} - Size: {} -- Current Size: {}", name, size / MB, currentSize.sum() / MB);
         }
     }
 
     boolean hasSpaceInCache() {
-        long currentSize = this.currentSize.get();
+        long currentSize = this.currentSize.sum();
```

Comment:
This operation will become more expensive. If we're checking this everything, it might not be good to do it with `LongAdder`

Reply:
Yes, I was wondering the same thing @merlimat. Many of the calls are from the managed ledger thread, so there might not actually be that much contention. It might help to provide more evidence that this is an optimization.

Reply:
+1